src/components/MerchantsPage/gam/styles.module.scss

```diff
@@ -176,3 +183,115 @@
     font-size: 16px;
   }
 }
+
+.noCampaign {
+  align-items: center;
+  border: 1px solid #e5e5e5;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  padding: 30px;
+
+  @media (max-width: 900px) {
+    grid-template-columns: 1fr;
+    margin-bottom: 15px;
+
+    & > div {
+      margin-bottom: 20px;
+    }
+  }
+}
+
+.noCampaignText {
+  :first-child {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
```

Comment:
To help reduce the size of css files in the future you might could refactor a lot of these font stylings into scss mixins